frontend/src/admin/components/sidebar/dropdown/dropdown-menu-item.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import "./dropdown-menu-item.scss";
+
+interface DropdownMenuProps {
+    id: string,
+    name: string
+}
+
+export default function DropdownMenuItem(props: DropdownMenuProps) {
```

Comment:
I think this component is taking it one step too far in separation of components. We should have a talk about it

Reply:
Yes it is indeed, this is removed and the items are children in the dropdown menu